### PR TITLE
dev template label update

### LINKED ISSUE #4478: Update labels: Issue Template: pre-work-template--dev
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] #4682
- [x] Replace the following line
   ```
   labels: 'Feature: Onboarding/Contributing.md, prework, size: 1pt'
   ```
   with
   ```
   labels: ['Feature: Onboarding/Contributing.md', 'prework', 'size: 1pt', 'role missing', 'Complexity: Prework']
   ```
- [x] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931).  Copy the contents of that comment into a new comment in this issue, and check off the boxes as you complete the steps.  

**For PR Reviewers and Merge Team**
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

**Link for Reviewers**
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository:  [REPLACE THIS TEXT AND BRACKETS WITH THE URL]

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/pre-work-template--dev.md`
- This issue is part of epic #4307

### PR DESCRIPTION
**Pull request by whitneywind**
Fixes #4478 

### What changes did you make?
  - updated labels in ".github/ISSUE_TEMPLATE/pre-work-template--dev.md"
 

For PR Reviewers and Merge Team
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

Link for Reviewers. - UPDATED 6/29
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository: https://github.com/whitneywind/website/issues/new?assignees=&labels=Feature%3A+Onboarding%2FContributing.md%2Cprework%2Csize%3A+1pt%2Crole+missing%2CComplexity%3A+Prework&projects=&template=pre-work-template--dev.md&title=Pre-work+Checklist%3A+Developer%3A+%5Breplace+brackets+with+your+name%5D

For PM, Merge Team, or Tech Lead

 Once the pull request associated with this issue is approved and merged, please update and edit epic https://github.com/hackforla/website/issues/4307 by
 Checking off the dependency for this issue
 If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### CODE REVIEW

**Comment by one2code** [comment]
**ETA:** 6/25/23

**Review by one2code — DISMISSED** [review]
Good naming convention for the issue branch, and the requested branch to merge into is correct. I've also verified that after clicking the link provided by @whitneywind, all of the updated labels correctly display by default.

Good job on your pull request and excellent attention to detail!

**Comment by roslynwythe** [comment]
Hi @whitneywind since this issue was written, there have been some changes to the labelling of prework issues, and so we may need to change the required `labels` string.  I will confirm with PM and report back on 6/27.  I apologize for the delay.

File: .github/ISSUE_TEMPLATE/pre-work-template--dev.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 name: Pre-work Template - Developers
 about: New developer team members should start by making this issue for themselves
 title: 'Pre-work Checklist: Developer: [replace brackets with your name]'
-labels: 'Complexity: Prework, Feature: Onboarding/Contributing.md, prework, size:
-  1pt'
+labels: ['Feature: Onboarding/Contributing.md', 'prework', 'size: 1pt', 'role missing', 'Complexity: Missing', 'Draft']
```
Comment (t-will-gillis) [inline .github/ISSUE_TEMPLATE/pre-work-template--dev.md]: Hello @whitneywind - My apologies, I gave you incorrect information previously. We **do** want to retain both the 'Complexity: Prework' and the 'prework' labels. Therefore, the edit should be: 

labels: ['Feature: Onboarding/Contributing.md',  'Complexity: Prework',  'prework', 'size: 1pt', 'role missing', 'Draft']

**Review by t-will-gillis — CHANGES_REQUESTED** [review]
Hi @whitneywind - My apologies- I gave you the wrong information. This line should be changed to:
```
labels: ['Feature: Onboarding/Contributing.md', 'Complexity: Prework', 'prework', 'size: 1pt', 'role missing',  'Draft']
```
and not what I told you previously. Sorry for giving you incorrect info!

**Comment by one2code** [comment]
I'll be sure to update my review to account for the changes.

**Review by one2code — DISMISSED** [review]
@whitneywind I've reviewed the latest changes, and have verified that you have updated the labels to @t-will-gillis latest comment. Excellent job on quickly updating the labels to the needs of the project.

**Review by roslynwythe — CHANGES_REQUESTED** [review]
Hi @whitneywind I just realized that I need to request another change in the labels.  I have updated the Action Items on the issue to omit the `Draft` label.  You will have to push a new commit to your fork, re-check the Action Item checkbox and click the "re-review" icon in the PR.  I apologize for missing this problem in my first review, and I thank you for your patience.

**Review by t-will-gillis — APPROVED** [review]
Hi @whitneywind Great job! I am seeing the correct labels applied on the link you provided. Otherwise, your branches are correct, your description is good. Thank you for your patience!

**Review by one2code — APPROVED** [review]

**Review by roslynwythe — APPROVED** [review]
Thank you @whitneywind for your excellent work on this issue and for making the requested changes.  I do apologize that we did not recognize the problem with the issue until you had submitted the PR.

**Pull request merged**